orchagent/orch.h

```diff
@@ -295,6 +295,16 @@ class Orch
     virtual void doTask(swss::NotificationConsumer &consumer) { }
     virtual void doTask(swss::SelectableTimer &timer) { }
 
+    /*
+     * Called once after APPLY_VIEW in warm/fast boot scenario.
+     * Orch can override this method to perform orch specific operations after boot is finished.
+     * These operations are not meant to produce additional ASIC configuration,
+     * instead a capability fetch and STATE_DB update here is encouraged.
+     * Orch is not expected to call the base method implementation as it must remain
```

Comment:
> These operations are not meant to produce additional ASIC configuration. instead a capability fetch and STATE_DB update here is encouraged

Is this change in expectation? What happens (before and after this change) when these operations result in additional ASIC configuration?

Also, how do you propose to enforce this expectation and add tests to prevent this?
When there is true capability change (from an old SAI/SDK to new, for eg.) across warm/fast reboot then how to still take that into effect?